README.md

```diff
@@ -1,23 +1,21 @@
 <a href="https://tiledb.com"><img src="https://github.com/TileDB-Inc/TileDB/raw/dev/doc/source/_static/tiledb-logo_color_no_margin_@4x.png" alt="TileDB logo" width="400"></a>
 
-
-
 # TileDB Vector Search
 
-*TileDB-Vector-Search* is a C++ library and Python API for vector search built on top of the [TileDB Storage Engine](https://github.com/TileDB-Inc/TileDB).
+_TileDB-Vector-Search_ is a C++ library and Python API for vector search built on top of the [TileDB Storage Engine](https://github.com/TileDB-Inc/TileDB).
```

Comment:
Is this intended?

Reply:
Not particularly, but it appears they both result in italics when viewed as a markdown preview, and it seems to be something you can't change about prettier: https://stackoverflow.com/questions/64052818/prettier-keeps-replacing-asterisk-characters-with-underscores